### ISSUE TITLE
Problem setting up a classroom repository

### ISSUE THREAD

**Issue by ariyanzri** [education/classroom#2343]
**Describe the bug**
I cannot join a classroom repository under the address [Link](https://classroom.github.com/a/CXAdCXKm ). When it is trying to setup the repository for me, it prompts with the error message "Assignment could not be created, please try again. (Validation failed: Assignment Should only have one assignment repository for each user-assignment combination)". I need to note that Github has restricted my account apparently because I'm from Iran and have traveled there for the summer. 

**Screenshots**
![test](https://user-images.githubusercontent.com/44510322/64365611-63b9ae00-cfc9-11e9-8b88-03c0400bbf25.png)

@bethard

**Comment by spinecone** [comment]
Hi @ariyanzri , thanks for reaching out! Since this is related to your github.com account being restricted, please contact  `developersupport@github.com` and they can work with you to get to the bottom of this issue.

**Comment by bethard** [comment]
Is it definitely the case that the problem is the GitHub.com account being restricted? The GitHub Classroom message makes it look like it's a different problem. (But maybe that's just a messaging error?)

**Comment by spinecone** [comment]
Hi @bethard , we can investigate the classroom error since it may be a separate problem, but we aren't able to help with the account restriction from our side, and continuing to use a account that's been restricted will likely cause other problems.

**Comment by bethard** [comment]
Yeah, if account restriction is the problem, it would be nicer if the error message said something like that. Looking a bit at what GitHub has done to Iranian developers, I'm guessing the message should say something like "Your account is unable to create private repositories"?

**Comment by spinecone** [comment]
Hi @ariyanzri , I've heard from our appeals review team that you submitted an appeal to reactivate your account and that it was successful. Have you experienced any more trouble with Classroom since then?

**Comment by ariyanzri** [comment]
Hi. Yes they reactivated my account yesterday but since I hadn't heard from them, I created another account and joined the classroom using the new account.
